core/controllers/base_test.py

```diff
@@ -200,7 +200,7 @@ def test_redirect_in_logged_out_states(self):
         """Test for a redirect in logged out state on '/'."""
 
         response = self.get_html_response('/', expected_status_int=302)
-        self.assertIn('splash', response.headers['location'])
+        self.assertEqual(len(response.headers['location'].split('/')[-1]), 0)
```

Comment:
This seems too complicated. Can you do a simpler check? Assert that it ends with / perhaps and that there's the right number of parts in the URL?

Reply:
Done.